### ISSUE TITLE
IP: Accessibility and WCAG Compliance

### ISSUE THREAD

**Issue by maxvisser** [teamforus/general#253]
## Tasks
- [ ] Use HTML5 elements instead of mainly using DIVs for example: 
   - Change all text links, buttons which are now using DIV-elements to button-elements
@aghimpu 

- [ ] add alt texts to all img-elements to describe whats on the image, if its an image which has only decorative purpose add an empty alt="" 
@SjoerdHilhorst 

note: user should be able to do everything with only a keyboard

- [ ] Role / ID attributes to make relations html elements more clear (ARIA)
@aghimpu 

- [ ] Scan the website on contrast ratio

- [ ] check keyboard accessibility, for example:
  -  apply only using a keyboard to a fund
  - open a provider details page using only your keyboard

**Comment by SjoerdHilhorst** [comment]
https://docs.google.com/document/d/1OEJsBrF6JSZwGeNZA4OXPbIViBBAs_24ZcwLSTv1uo0/edit?usp=sharing

**Comment by GerbenBosschieter** [comment]
Removed my assignment untill we decide to include this in our pipeline and review is being done based on the guidelines.
We can find the guidelines here:

https://www.w3.org/TR/WCAG20-TECHS/Overview.html#contents

**Comment by jamalv** [comment]
got audited, recieved a report, will update this issue

**Comment by maartenfv** [comment]
@jamalv @GerbenBosschieter wat is de status van de WCAG compliancy?

**Comment by maartenfv** [comment]
Doel is om voor Groningen ASAP compliant te zijn en de verklaring te updaten. Ook belangrijk voor aankomende opdrachten.

**Comment by jamalv** [comment]
qua takenverdeling is mijn beeld:
- bijna alles staat in review dus max/gerben. gerben heeft onlangs hier een update gegeven: https://discord.com/channels/708583667634667560/708588070739509248/842051517766172712
- een aantal staan nog in de backlog, die zijn vooral gerelateerd aan design dus afhankelijk van hoe ver dat is, is tijd van linda, alex en/of sjoerd benodigd
- afhankelijk van review resultaten nog ontwikkeltijd van alex/sjoerd
- zodra het gereleased is of we een motivatie hebben waarom bepaalde issues niet verhopen worden (dat kan natuurlijk), kunnen we een hertest aanvragen bij janita

**Comment by jamalv** [comment]
van de 84 issues:
30 gesloten
46 in review
7 op de backlog
1 in de icebox

**Comment by maartenfv** [comment]
Besproken met Linda ze kan dit vanaf woensdag oppakken.

**Comment by jamalv** [comment]
Gerben geeft aan issues die nu in review staan lijken niet te zijn opgelost.

Maarten geeft aan graag meeting plannen over stand van zaken, @jamalv schiet wat in.

**Comment by jamalv** [comment]
straks tijdens refinement bespreken, morgen update

**Comment by jamalv** [comment]
update tijdens refinement was dat de meeste issues ter review klaarliggen, er is tijd van max nodig om er doorheen te lopen

**Comment by jamalv** [comment]
max heeft er nog niet naar gekeken; 

max gaat kijken:
- wat staat nog open?
- wat is er nodig om het af te maken? quick wins/wat is groter?

**Comment by jamalv** [comment]
gerben heeft ook wat tijd gereserveerd

**Comment by jamalv** [comment]
gerben en max spelen door naar alex en sjoerd

**Comment by jamalv** [comment]
max heeft er gister best veel tijd in gestopt; angular heeft zelf een aria dependency die we niet gebruiken; die voegt voor de meest gebruikte dingen ARIA labels toe en keyboard toegankelijkheid. Die heeft max als dependency toegevoegd, ook even met Valentin besproken. Scheelt hopelijk tijd in de toekomst.

**Comment by jamalv** [comment]
gerben gaat nog even een review doen van wat max heeft laten zien; plan is om het mee te nemen in de release van maandag en ook opnieuw de test te doen en issues te doorlopen.

**Comment by jamalv** [comment]
release staat voor morgen geplanned

**Comment by jamalv** [comment]
is gereleased!

**Comment by jamalv** [comment]
gerben verwact vandaag begin, morgen verder

**Comment by GerbenBosschieter** [comment]
Nieuwe review gedaan, alle issues die opgelost zijn gesloten. Veel vooruitgang geboekt.

- 12 issues nog niet opgelost en moeten terug naar development.
- 7 issues waarvan ik niet zeker ben, opnieuw moet kijken of even moet navragen

**Comment by jamalv** [comment]
bal nu bij @maxvisser, die gaat het bij refinement aan alex geven

**Comment by jamalv** [comment]
gerben geeft aan nog even goed intern checken voordat we hertest doen.

**Comment by jamalv** [comment]
doel: voor volgende release zoveel mogelijk verwerken, zoveel mogelijk testen en daarna met janita weer een hertest. evt hotfix dingen

**Comment by jamalv** [comment]
issues zijn gister met refinement overgedragen aan Alex

**Comment by maartenfv** [comment]
Gerben geeft aan we houden ons aan de planning.

**Comment by maxvisser** [comment]
Eerst schermlezersoftware zelf testen, issues aanmaken en fixen daarna hertest!

**Comment by jamalv** [comment]
Alex fixt issues - gerben is op vakantie.

**Comment by jamalv** [comment]
gerben is bezig met screenreader dingen - het loopt!

**Comment by jamalv** [comment]
gerben gaat vanmiddag verder en hoopt nog wat aan de release te kunnen toevoegen

**Comment by jamalv** [comment]
Alex heeft een pr geopend - volgende week gaat dit verder

**Comment by jamalv** [comment]
vanochtend is weer een deel gereleased 

gerben verwacht vandaag/morgen pull request voor screenreader dingen

janitia wordt gecontacteerd door max zodra het klaar is.

**Comment by maxvisser** [comment]
gerben opent pull request binnenkort

**Comment by jamalv** [comment]
gerben; 'jaaaa, dit gaat lekker jongens' - pr staat klaar, gerben is nog uitgebreid aan het testen met tooling

interne tool van chrome geeft ons een 90+ score - we gaan heel lekker

zijn geen grote dingen meer

**Comment by jamalv** [comment]
gerben heeft dit weekend pr geopend - laatste lootjes - alle tools van janita is 100%

na daily even met max over de release - vervolgens kan janita ingepland worden

**Comment by jamalv** [comment]
vraag om na dat gesprek (zekerheid release) janita inplannen

**Comment by jamalv** [comment]
is besproken: commits staan in een branch, koop op afstand houdt het tegen omdat er veel pagina's met koop op afstand zijn geraakt.

plan is om koop op afstand en huidige fixes maandag te releasen - janita kan dan na komende maandag gevraagd worden.

**Comment by jamalv** [comment]
max kan bij janita gaan checken of/wanneer ze beschikbaar is (na maandag)

**Comment by jamalv** [comment]
na maandag heeft max waarschijnlijk ruimte hiervoor

vraag van gerben om rapport van janita te delen @jamalv

**Comment by jamalv** [comment]
gerben heeft rapport gelezen en dingen aangegeven die hij nog wil checken - focus ligt nu op release kan zijn dat er nog dingen naar boven komen.

**Comment by jamalv** [comment]
@GerbenBosschieter zet deze na de release weer op high prio

**Comment by jamalv** [comment]
release is er uit! - nu janita vragen?

@GerbenBosschieter zegt ; nog 1 verse test er doorheen en dan janita vragen

@maxvisser en @GerbenBosschieter hoe om te gaan met laatste fixes?

@maxvisser draagt vraag aan janita over naar gerben

**Comment by maxvisser** [comment]
zelfde status als gisteren.

Gerbens doel is om vandaag de fixes van WCAG klaar te hebben en evt al op staging te hebben staan.

Gerbens afhankelijkheid zit hem in context van hoe het de vorige keer ging en bespreekt dit graag met @jamalv

**Comment by GerbenBosschieter** [comment]
Mail naar Janita gestuurd met vraag waneer ze een indicatie heeft op welke termijn ze zou kunnen. LAatste fixes zijn nog niet doorgevoerd.

**Comment by jamalv** [comment]
janita test op staging - laatste fixes gaan binnenkort naar staging

prio is afhankelijk van janita's planning

**Comment by jamalv** [comment]
janita is begin september beschikbaar (week 38/39)
technisch vanochtend doorlopen, alleen nog aanvraagproces, kan geen issues meer vinden :D

**Comment by jamalv** [comment]
@GerbenBosschieter zet deze issue begin september weer op high prio!

**Comment by jamalv** [comment]
>  Ik heb deze keer de communicatie gedaan met Janita maar heb geen context van het financiele gedeelte @jamalv kun jij hiermee helpen of wanneer nodig overnemen?

sure! mijn verwachting is dat het maximaal €360 in beslag neemt (hertest + bespreking) als we near 100% zou bespreking niet eens nodig zijn. dus dan €270,- 

 ![Screenshot 2021-09-01 at 10.51.30.png](https://images.zenhubusercontent.com/5bc6fc7b86f3a678d497cf05/55f278a7-5515-4cea-93bd-ed37c47534cf)

dan is het kwestie van doen & factuur naar finance@forus.io

**Comment by maxvisser** [comment]
Aanvraagproces is eigenlijk niet te testen met de tools van Janita.

**Comment by maxvisser** [comment]
Max gaat gerben helpen bij het testen van fixes (door localhost te koppelen aan staging)

**Comment by jamalv** [comment]
gerben liep vast maar gaat nu weer de goede kant op - maandag janita!
vraag aan robin is om te stagen zodra pr er is

**Comment by jamalv** [comment]
mail naar janita is toegestuurd - loopt nu niet stuk

**Comment by jamalv** [comment]
gerben houd in de gaten en zet weer high prio wanneer nodig.

**Comment by jamalv** [comment]
actie bij @maxvisser om janita te bellen

**Comment by maxvisser** [comment]
Janita komt langs 11 januari

**Comment by maartenfv** [comment]
Please maak nieuwe isseu voor janita in itsm geen project meer.

**Comment by maartenfv** [comment]
Please maak nieuwe isseu voor janita in itsm geen project meer.
